app/queries/keywords_query.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class KeywordsQuery
+  attr_reader :keywords
+
+  def initialize(user)
+    @keywords = user.keywords
+  end
+
+  def call
+    @keywords = order
+  end
+
+  private
+
+  def order
```

Comment:
can be rename it to order_by_name. or even keeping it in same call function can also be ok for now.